docs/extend/extend-apiml/implementing-routing-to-the-api-gateway.md

```diff
@@ -1,7 +1,23 @@
 # Implementing routing to the API Gateway
 
+As an extender of API Mediation Layer, review how to implement basic routing to the API Gateway. 
```

Comment:
Also if we touch this area, we should provide a simple image to show the routing.

Reply:
Would you be able to provide the image to include here?